src/java.base/share/classes/java/lang/StringCoding.java

```diff
@@ -600,7 +606,7 @@ private static Result decodeLatin1(byte[] ba, int off, int len) {
     }
 
     @IntrinsicCandidate
-    private static int implEncodeISOArray(byte[] sa, int sp,
+    public static int implEncodeISOArray(byte[] sa, int sp,
```

Comment:
Is this change needed?

Reply:
Yes, this is to match `StringCoding` in `master` branch to be referred by `String`.